tests/test_signed_links.py

```
@@ -2,31 +2,31 @@
 def test_hmac_verify_expired() -> None:
-    secret = "x"
+    secret = "x"  # nosec B105
     path = "/api/x"
     exp = 100
     sig = sign(secret, path, exp)
     assert not verify(secret, path, exp, sig, now_ts=exp + 1)
 
 
 def test_hmac_verify_wrong_signature() -> None:
-    secret = "y"
+    secret = "y"  # nosec B105
```

Ruff (0.14.0)

Possible hardcoded password assigned to: "secret"

(S105)